lib/sd_notify.rb

```diff
@@ -104,21 +106,16 @@ def self.watchdog?
   def self.notify(state, unset_env=false)
     sock = ENV["NOTIFY_SOCKET"]
 
-    return nil if !sock
+    return nil unless sock
```

Comment:
:fire: I belong in the "no unless" camp :grimacing: :fire:

Reply:
Removed